easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.1'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
```

Comment:
You want toolchainopts = {'openmp': True} for both of them.

Reply:
Good point here. I noticed that that toolchain flag is not present in any `RELION` easyconfig. Maybe worth considering opening another PR that fixes this?

Reply:
One question. How does the addition of `-fopenmp` interacts with `-lpthread` in this context?

Reply:
As far as I have seen there is no problem.